clang/docs/analyzer/checkers.rst

```diff
@@ -1179,6 +1179,34 @@ security.insecureAPI.DeprecatedOrUnsafeBufferHandling (C)
    strncpy(buf, "a", 1); // warn
  }
 
+security.SetgidSetuidOrder (C)
+""""""""""""""""""""""""""""""
```

Comment:
Give that the [`alpha.unix.chroot`](https://clang.llvm.org/docs/analyzer/checkers.html#alpha-unix-chroot-c) checker does something similar, I wonder if this checker should share the same parent package with that one to aid discoverability.
WDYT?

Reply:
Move the new checker into `unix`, or move the chroot checker into `security` (or `alpha.security`)?

Reply:
I don't have a preference, I was just asking.
I just felt this is a natural question, and you seem to care about alpha checkers and documentation/discoverability, and this seemed like a subject to discuss. @NagyDonat WDYT?